open-company/open-company-email
# Storage lib

Card: https://trello.com/c/Iy7UkiLu

### Review after https://github.com/open-company/open-company-lib/pull/45

Related PRs:
- [Lib](https://github.com/open-company/open-company-lib/pull/46)
- [Storage](https://github.com/open-company/open-company-storage/pull/177)
- [Auth](https://github.com/open-company/open-company-auth/pull/69)
- [Bot](https://github.com/open-company/open-company-bot/pull/45)
- [Infra](https://github.com/belucid/open-company-infrastructure/pull/94)


To test:
- setup your Slack user to get notifications in slack
- setup an Email user to get notifications via email (make sure it doesn't have a Slack user connected)
- [x] add a post mentioning the Slack user in the body, did you get the Slack message with context? Good
- [x] comment on that post mentioning the Slack user, did you get the Slack message with context? Good
- now with the Slack user
- [x] add a post mentioning the Email user, did you get the email with context? Good
- [x] add a comment mentioning the Email user, did you get the email with context? Good
- [x] check that the email template is ok with the design
- update lib version removing alpha
- release lib version
- remove alpha from lib version from: storage, email and bot
- merge
- deploy
